code/modules/xenoarchaeology/xenoartifact_structure.dm

```diff
@@ -0,0 +1,277 @@
+/*
+	Pretty much a duplicate of the regular item.
+	Look at the item for comment documentation, most of the comments here are just artifacts from the copy paste
+	Use this file to house the weird off-shoot artifacts. 
+*/
+
+/obj/structure/xenoartifact
+	name = "xenoartifact"
+	icon = 'icons/obj/xenoarchaeology/xenoartifact.dmi'
+	icon_state = "map_editor"
+	density = TRUE
+	
+	var/charge = 0 //How much input the artifact is getting from activator traits
+	var/charge_req //This isn't a requirement anymore. This just affects how effective the charge is
+
+	var/material //Associated traits & colour
+	var/list/traits = list()
+	var/datum/xenoartifact_trait/touch_desc
+	var/special_desc = "The Xenoartifact is made from a" //used for special examine circumstance, science goggles
+	var/process_type
+	var/code //Used for signaler trait
+	var/frequency
+	var/datum/radio_frequency/radio_connection
+	var/min_desc //Just a holder for examine special_desc from minor traits
+
+	var/max_range = 1 //How far his little arms can reach
+	var/list/true_target = list()
+	var/usedwhen //holder for worldtime
+	var/cooldown = 8 SECONDS //Time between uses
+	var/cooldownmod = 0 //Extra time traits can add to the cooldown
+
+	var/icon_slots[4] //Associated with random sprite stuff, dw
```

Comment:
[4]

Reply:
Changed to list. Numbering it is important to trait faces